test/tests/nextcloud-apache-run/run.sh

```diff
@@ -23,7 +23,7 @@ _request() {
 }
 
 # Make sure that Apache is listening and ready
-. "$dir/../../retry.sh" --tries 30 '_request GET / --output /dev/null'
+. "$dir/../../retry.sh" --tries 50 --sleep 5 '_request GET / --output /dev/null'
```

Comment:
From 1 minute to over 4; that seems excessive with 4 tests on each of up to 18 distinct images (5 tests, but only 4 are used on each specific image).